packages/wonder-blocks-banner/src/components/banner.tsx

```diff
@@ -324,41 +322,44 @@ const Banner = (props: Props): React.ReactElement => {
 const bannerTokens = {
     root: {
         border: {
-            radius: {
-                default: border.radius.radius_0,
-                floating: border.radius.radius_040,
```

Comment:
We no longer need to support the different banner layouts so we can flatten this structure! We use the `theme` because we use different radius values for Classic and TB

Reply:
For reference later on, we discussed how we want to use static component tokens in https://github.com/Khan/wonder-blocks/pull/2655#discussion_r2132852528 !